### DISCUSSION TITLE
Auto level problem

### DISCUSSION THREAD

**Discussion by wiebster112** [Jyers/Marlin#1675] [Q&A]
hey,
my name is Wiebster and I got a problem whenever I auto-level and print something the left side is always higher than the left side so much i cant print anything. I have tried making multiple meshes I have manually leveled the bed (whit paper and with the sensor) but the left side always stays higher
I used the 5x5 newest firmware for the 4.2.2 mainboard
Also tried the 3x3 firmware
please help me

Kind regards,
Wiebster

**Comment by Sirnut** [comment]
I had a similar experience and found adding more clips helped. I have each corner clipped now instead of just the two that came with the printer

**Comment by PapsDizzy** [comment]
befor creating a mesh you must level your bed by hand and you musst adjust ur z offset by hand. then you can go on and create a mesh that will work fine
